torch/csrc/jit/passes/quantization.cpp

```diff
@@ -2079,6 +2191,7 @@ void QuantFusion(std::shared_ptr<Graph>& graph) {
 Module FoldConvBatchNorm2d(const Module& module) {
   FoldConvBatchNorm2dHelper h;
   Module m = module.clone();
+  addBiasForConv2dIfNone(m);
```

Comment:
can you put this around here: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/quantization.cpp#L816?

The plan is we'll separate these functions into a separate preprocess pass that will align tracing and scripting, we'll later have a PR for that.

Reply:
For 1) that will miss the cases I want to catch because now I have to do this preprocessing for batch norm fusion. But more importantly I dont think it belongs there. It is a preprocessing that is strongly coupled with FoldConvBatchNorm2d and should be as close to it as possible, such that it is independently callable.

Reply:
OK, makes sense